packages/dd-trace/src/plugins/ci_plugin.js

```diff
@@ -111,7 +111,10 @@ module.exports = class CiPlugin extends Plugin {
     const childOf = getTestParentSpan(this.tracer)
 
     let testTags = {
-      ...getTestCommonTags(testName, testSuite, this.frameworkVersion),
+      ...getTestCommonTags(testName,
+        testSuite,
+        this.frameworkVersion,
+        this.constructor.id),
```

Comment:
we don't force any style, but this is more in line with the rest of the code
```suggestion
      ...getTestCommonTags(
        testName,
        testSuite,
        this.frameworkVersion,
        this.constructor.id
      ),
```